test/javascript/util/core.js

```diff
@@ -92,9 +92,8 @@ describe('Foundation core', function() {
       name.should.be.equal('A');
     });
 
-    it('should handle a function expression', function() {
-      var B = function(){}; 
-      var name = Foundation.getFnName(B);
+    it('should handle an anonymous function expression', function() {
```

Comment:
Could you add a comment about the "inconsistent behavior" ?
Like:
```
// Inferred names (i.e. `var name = function() {}`) are not tested as they are widely supported
// See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name#Inferred_function_names
```

Reply:
Done.